scripts/joint_trajectory_file_playback.py

```diff
@@ -233,12 +239,38 @@ def find_start_offset(pos):
             cur_cmd = [cmd['right_gripper']]
             self._add_point(cur_cmd, 'right_gripper', values[0] + start_offset)
 
+    def _feedback(self, data):
+        if not self._get_trajectory_flag():
+            self._set_trajectory_flag(value=True)
+
+    def _set_trajectory_flag(self, value=False):
+        # Lock
+        self._lock.acquire()
+        # Assign a value to the flag
+        self._arm_trajectory_started = value
```

Comment:
Yeah, it seems `deepcopy()` only matters for compound objects. I believe simply assigning value would suffice since we are just dealing with bools: https://docs.python.org/2/library/copy.html